frontend/src/components/maps/leaflet/InfoSlideOut/InfoContent.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { ListGroup } from 'react-bootstrap';
+import { IBuilding, IParcel, PropertyTypes } from 'actions/parcelsActions';
+import { Label } from 'components/common/Label';
+import { ParcelPIDPIN } from './ParcelPIDPIN';
+import ParcelAttributes from './ParcelAttributes';
+import './InfoSlideOut.scss';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import Claims from 'constants/claims';
+
+export type IInfoContent = {
+  /** the selected property information */
+  propertyInfo: IParcel | IBuilding | null;
+  /** The property type [Parcel, Building] */
+  propertyTypeId: PropertyTypes | null;
+};
+
+/**
+ * Component that displays the appropriate information about the selected property
+ * in the property info slideout
+ * @param propertyInfo the selected property
```

Comment:
slightly misleading doc since this component takes in a single param, props, with a number of fields. One option is to do this:

```
interface ITenancyProps {
  title: string;
  disabled?: boolean;
  nameSpace?: string;
  index?: any;
  showImprovements?: boolean;
}

/**
 * Display valuation tables for assessed and netbook values.
 * @param {ITenancyProps} param0
 */
```

Reply:
ooooh I getcha